src/pages/billing/List.js

```diff
@@ -271,6 +279,7 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
   useOnMount(() => {
     loadProjects()
     loadAccounts()
+    loadAlphaSpendReportMember()
```

Comment:
I put the loading of this value in this file so that we don't make the Ajax call every time a project is loaded (since this value does not change by project).